DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -52,5 +53,13 @@ public IEnumerable<DelegateCourseDetails> GetAllCoursesForDelegate(int delegateI
                 }
             );
         }
+
+        public bool VerifyUserCanAccessCourse(int customisationId, int centreId, int categoryId)
+        {
+            var (courseCentreId, courseCategoryId) =
+                courseDataService.GetCentreIdAndCategoryIdForCourse(customisationId);
+
+            return centreId == courseCentreId && (categoryId == courseCategoryId || categoryId == 0);

```

Comment:
Just want to double check something: some customisations have the AllCentres meaning that a user can be enrolled on the customisation despite having a different centreId for that record, is that taken into account here? I think it should be fine as users aren't allowed to edit AllCentre customisations unless they are at the same centre.

Reply:
Ah, yeah I think that doesn't apply here because we're only concerned about AdminUsers, the ones who can edit course admin fields. I've changed the name of the VerifyUserCanAccessCourse method to VerifyAdminUserCanAccessCourse for clarity